### DISCUSSION TITLE
Artificiel horizon, add parallel line

### DISCUSSION THREAD

**Discussion by imotep33** [iNavFlight/inav-configurator#2279] [Ideas]
In usual planes artificial horizon, there are more lines than just the main horizon. It would be nice to be able to have more than 1 line. and to be able to configure the space between those bars.
My main concern is when I clim up with my wing, the camera points the sky and I don't really know if my trajectory is straight or if I roll to one side. I tried with the pitch and roll angle, but it's not ergonomic at all and by the time I understand my position, it's too late to correct.
![image](https://github.com/user-attachments/assets/072afa3a-f2ed-4765-b55d-8c2513d8f5b4)

**Comment by MrD-RC** [comment]
You can have this. If you use the FrSky PixelOSD. To do this effectively, it needs to use the canvas draw method. Most systems don't. Despite some digital systems, like DJI, claiming "canvas OSD". It isn't. The only OSD that has ever use true canvas mode with INAV is the FrSky PixelOSD. 

This does exist. https://github.com/iNavFlight/inav/blob/master/docs/Settings.md#osd_ahi_pitch_interval But it is not going to give details like the degrees of pitch, like the actual AHI in manned aircraft.

**Comment by MrD-RC** [comment]
The easiest way to tell if you're flying level is to use the digital vario (will be renamed VSI in the future). This gives the rate of climb or descent. The AHI may not be level with the horizon, unless you have tuned it. It's based on the camera angle.

**Comment by sensei-hacker** [comment]
I just want to re-emphasize osd_ahi_pitch_interval which Mr D pointed out. So that information, which I think answers request, doesn't get lost in the conversation about canvas mode etc.

@imotep33 do this in the CLI tab:

set osd_ahi_pitch_interval = 15
save

**Reply by imotep33** [reply]
@sensei-hacker I was testing it, that's what I was looking for! need to tune the right amount of angle now.
@MrD-RC  I already use the vario, but it's acceptable only for light pitch angle. I was really looking for someting to help me understand how the craft is when it's about 60-90° angle.

Thank you for the answer.
